sources/router.py

```
@@ -523,7 +525,10 @@ def select_agent(self, text: str) -> Agent:
         "给我讲一个有趣的故事",
         "Raconte moi une histoire drole"
     ]
-    for text in texts:
-        print("Input text:", text)
-        agent = router.select_agent(text)
-        print()
+    async def main():
+        for text in texts:
+            print("Input text:", text)
+            agent = await router.select_agent(text)
```

Ruff (0.15.0)

[error] Local variable `agent` is assigned to but never used

Remove assignment to unused variable `agent`

(F841)